Cargo.toml

```diff
@@ -32,6 +32,8 @@ rayon = "1.3.0"
 memmap = "0.7.0"
 thiserror = "1.0.10"
 ahash = "0.3.4"
+structopt = { version = "0.3", default-features = false }
+env_logger = "0.7.1"
```

Comment:
These dependencies should be optional, or even dev-dependencies. We don't want folks building this when they use bellman as a dependency